templates/account/signup.html

```diff
@@ -41,8 +43,10 @@ <h3 class="welcome">Welcome!</h3>
                     {% endif %}
                     <button class="signupbutton" type="submit">{% trans "Sign Up" %}</button>
                     <br>
+    
                 </div>
             </form>
+            <a href="{% provider_login_url 'google' %}"><button class="googlebutton">Sign Up with Google</button></a>
```

Comment:
It would be complex to test google auth from pr deploys, since it is not listed in the auth callbacks
@maneprajakta What do you think

Reply:
@Praful932 yes we can add only one URL in there. I think this will work.